src/common/ceph_context.h

```diff
@@ -306,7 +304,7 @@ class CephContext {
 
   int _crypto_inited;
 
-  std::shared_ptr<std::vector<entity_addrvec_t>> _mon_addrs;
+  std::atomic<std::shared_ptr<std::vector<entity_addrvec_t>>> _mon_addrs;
```

Comment:
This wasn't available yet:

https://en.cppreference.com/w/cpp/memory/shared_ptr/atomic2

but now is (for Ceph). LGTM